<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="TKonstantinov" name="Command line test" tests="1" failures="1" timestamp="14 Oct 2013 15:11:51 GMT" time="1.043" errors="0">
  <testcase name="main" time="1.025" classname="test.test.rrrTest">
    <failure type="java.lang.RuntimeException" message="Could not contact Selenium Server; have you started it on &amp;apos;localhost:4444&amp;apos; ?
Read more at http://seleniumhq.org/projects/remote-control/not-started.html
Connection refused: connect">
      <![CDATA[java.lang.RuntimeException: Could not contact Selenium Server; have you started it on 'localhost:4444' ?
Read more at http://seleniumhq.org/projects/remote-control/not-started.html
Connection refused: connect
	at com.thoughtworks.selenium.DefaultSelenium.start(DefaultSelenium.java:104)
	at test.test.rrrTest.testGoogle(rrrTest.java:22)
	at test.test.rrrTest.main(rrrTest.java:13)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
... Removed 25 stack frames]]>
    </failure>
  </testcase>
</testsuite>
